### DISCUSSION TITLE
Make export templates downloadable independently

### DISCUSSION THREAD

**Discussion by bart-tek** [godotengine/godot-proposals#6327] [General Discussions]
Hello, I was wondering if it would be possible to make the different export templates dowloadable independently on the download repository. 

Currently, the .tpz file contains the web, ios, android, windows, macos and linux templates, for a size of about 800 Mo (for Godot 4.0 rc2). In our project, we have automated the building of exports by using a custom docker image (based on [abarichello's ](https://github.com/abarichello/godot-ci), but adapted for Godot 4.0). This image needs to be rebuilt each time a new version of Godot 4.0 is released, and we have to download the export templates all over again, which can sometimes be bothersome as our connection is not always the greatest (fluctuating between 100 Ko/s and 1 Mo/s). Since we only need the web template, it would be quite helpful if we could download only this one.

Thank you for considering this request.

**Comment by Calinou** [comment]
This is already being tracked in https://github.com/godotengine/godot-proposals/issues/647.
